index.html

```diff
@@ -884,7 +884,30 @@ <h2>The <code>application/credential-claims-set-1.1+json</code> Media Type</h2>
   </table>
   <div class="issue" data-number="42"></div>
 </section>
+</section>
 
+<section id="appendix">
+  <h2>Appendix</h2>
+  <section>
+    <h3>Example VC Data Model Mapping</h3>
+    <p>This is one possible unidirectional mapping between 2.0 VC-JWTs and the VC Data Model; other such mappings are possible.</p>
```

Comment:
This is an invalid mapping.

The mapping is non-conformant with the VC Data Model specification because `credentialSubject` can't be a string: https://w3c.github.io/vc-data-model/#credential-subject as processed via this issue: https://github.com/w3c/vc-data-model/issues/762 and resolved by consensus in this PR: https://github.com/w3c/vc-data-model/pull/970 and this PR: https://github.com/w3c/vc-data-model/pull/976